### ISSUE TITLE
geo nuke_placer option hasn't done anything since 1.5.0

### ISSUE THREAD

**Issue by tsteven4** [gpsbabel/gpsbabel#1350]
I think this worked in 1.4.4.  The documentation states:
```
nuke_placer option
Omit Placer name.

If this option is specified, GPSBabel will not read geocache placer information from a .loc file on input. That is, it will ignore any placer names in the input file.

This option has no effect on output.
```

Should we restore the functionality or delete the option?

**Comment by robertlipe** [comment]
Delete it. Nobody noticed, so nobody's using it.

**Comment by tsteven4** [comment]
This was resolved in #1355
